docs/data-sources/kubernetes_secret.md

```diff
@@ -0,0 +1,87 @@
+---
+Title: "Secret Resource"
+Description: |-
+    Fetching the details of sceret resource.
+---
+
+# Namespace
+
+Manage a sceret using this Terraform module.
```

Comment:
Please get this content reviewed by the feature owners. They can suggest better what needs to be added. Also @ishangupta-ds can take a look at this.

Reply:
@ishangupta-ds PTAL

Reply:
Left a few comments.